concourse/pipelines/templates/jobs/test-tpl.yml

```
@@ -26,6 +26,9 @@
     {# non-cloud impersonation tests (regular) run a standard group combination #}
     {% set test_group = 'gpdb,proxy,profile' %}
 {% endif %}
+{% if extension == 'fdw' %}
```

```suggestion
{% if x.use_fdw == 'true' %}
```
since `x.use_fdw` is the primary source of info while `extension` is a derived value to use in the job name only